internal/pkg/diagnostics/diagnostics.go

```diff
@@ -507,7 +507,12 @@ func zipLogsWithPath(pathsHome, commitName string, collectServices, excludeEvent
 			return nil
 		}
 
-		return saveLogs(name, path, zw)
+		// Add the file to the zip.
```

Comment:
nit: I am not allowed to comment on unchanged lines, but if you can please handle the linter issue above 🙂